README.md

```diff
@@ -42,192 +44,104 @@ After you're finished please remove all the comments and instructions!
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
```

Comment:
Nice README 💯 but seems like some sections were removed in the `Getting Started` part 👀 some may be removed as they are not relevant to the project for now but some others such as `Setup`, `Prerequisites` and `Install` should be there.